### PR TITLE
cpupower: add page

### PR DESCRIPTION
**Pull request by toydotgame**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 5.17-1

### CODE REVIEW

File: pages/linux/cpupower.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Tools regarding CPU power and tuning options.
+> This command is available as part of the `cpupower` package, or on Fedora as part of `kernel-tools`.
+> More information: <https://linux.die.net/man/1/cpupower>.
+
+- List CPUs:
+
+`sudo cpupower --cpu cpulist`
+
+- Get info on all cores:
```
Comment (navarroaxel) [inline pages/linux/cpupower.md:11]: We don't use abbreviations in example descriptions
 ```suggestion
- Print information about all cores:
```

File: pages/linux/cpupower.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Set all CPUs to a power-saving frequency governor:
+
+`sudo cpupower --cpu all frequency-set --governor powersave`
+
+- Get CPU 0's available frequency [g]overnors:
+
+`sudo cpupower --cpu 0 frequency-info g | grep "analyzing\|governors"`
+
+- Get CPU 4's frequency from the hardware, in human-readable form:
```
Comment (navarroaxel) [inline pages/linux/cpupower.md:23]:
```suggestion
- Print CPU 4's frequency from the hardware, in human-readable form:
```

File: pages/linux/cpupower.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Get info on all cores:
+
+`sudo cpupower --cpu all info`
+
+- Set all CPUs to a power-saving frequency governor:
+
+`sudo cpupower --cpu all frequency-set --governor powersave`
+
+- Get CPU 0's available frequency [g]overnors:
```
Comment (navarroaxel) [inline pages/linux/cpupower.md:19]:
```suggestion
- Print CPU 0's available frequency [g]overnors:
```

File: pages/linux/cpupower.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+> This command is available as part of the `cpupower` package, or on Fedora as part of `kernel-tools`.
+> More information: <https://linux.die.net/man/1/cpupower>.
+
+- List CPUs:
+
+`sudo cpupower --cpu cpulist`
+
+- Print information about all cores:
+
+`sudo cpupower --cpu all info`
```
Comment (blueskyson) [inline pages/linux/cpupower.md:13]:
```suggestion
`sudo cpupower --cpu {{all}} info`
```

File: pages/linux/cpupower.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Set all CPUs to a power-saving frequency governor:
+
+`sudo cpupower --cpu all frequency-set --governor powersave`
```
Comment (blueskyson) [inline pages/linux/cpupower.md:17]:
```suggestion
`sudo cpupower --cpu {{all}} frequency-set --governor {{powersave}}`
```

File: pages/linux/cpupower.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+- Print CPU 0's available frequency [g]overnors:
+
+`sudo cpupower --cpu 0 frequency-info g | grep "analyzing\|governors"`
```
Comment (blueskyson) [inline pages/linux/cpupower.md:21]:
```suggestion
`sudo cpupower --cpu {{0}} frequency-info g | grep "analyzing\|governors"`
```

File: pages/linux/cpupower.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+- Print CPU 4's frequency from the hardware, in human-readable form:
+
+`sudo cpupower --cpu 4 frequency-info --hwfreq --human`
```
Comment (blueskyson) [inline pages/linux/cpupower.md:25]:
```suggestion
`sudo cpupower --cpu {{4}} frequency-info --hwfreq --human`
```

File: pages/linux/cpupower.md (hunk 4 above)
Comment (blueskyson) [reply pages/linux/cpupower.md:13]: It means user provided value. See [token syntax](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#token-syntax).

Comment (toydotgame) [reply pages/linux/cpupower.md]: Ah, I see.
image

File: pages/linux/cpupower.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# cpupower
+
+> Tools regarding CPU power and tuning options.
+> This command is available as part of the `cpupower` package, or on Fedora as part of `kernel-tools`.
+> More information: <https://linux.die.net/man/1/cpupower>.
+
+- List CPUs:
```
Comment (CleanMachine1) [inline pages/linux/cpupower.md:7]: When I run this command, I get
Error parsing cpu list

Arch Linux

Is this the case on other distros?

File: pages/linux/cpupower.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+# cpupower
+
+> Tools regarding CPU power and tuning options.
+> This command is available as part of the `cpupower` package, or on Fedora as part of `kernel-tools`.
+> More information: <https://linux.die.net/man/1/cpupower>.
```
Comment (CleanMachine1) [inline pages/linux/cpupower.md:5]:
```suggestion
> More information: <https://manned.org/cpupower>.
```

File: pages/linux/cpupower.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+- Set all CPUs to a power-saving frequency governor:
+
+`sudo cpupower --cpu {{all}} frequency-set --governor {{powersave}}`
+
+- Print CPU 0's available frequency [g]overnors:
+
+`sudo cpupower --cpu {{0}} frequency-info g | grep "analyzing\|governors"`
+
+- Print CPU 4's frequency from the hardware, in human-readable form:
```
Comment (CleanMachine1) [inline pages/linux/cpupower.md:23]:
```suggestion
- Print CPU 4's frequency from the hardware, in a human-readable format:
```

File: pages/linux/cpupower.md (hunk 8 above)
Comment (toydotgame) [reply pages/linux/cpupower.md:7]: I run Arch too and I've never seen that error. Alternative commands like `sudo cpupower -c all info` (and more verbose: `sudo cpupower -c all frequency-info`) exist and do list the number of each CPU.
image

Would it be worth/should I replace `sudo cpupower --cpu cpulist` in the examples with `sudo cpupower --cpu all info`?

**Comment by toydotgame** [comment]
Damnit I broke it…

**Comment by toydotgame** [comment]
There's one commit on my repo ahead of this one, but it won't show up in this PR. I might make a new PR.

**Comment by navarroaxel** [comment]
@toydotgame not required, but yeah, create a new PR!

**Pull request closed without merging**